lib/manifest.js

```diff
@@ -290,6 +290,20 @@ internals.manifest = [
         list: true,
         useFilename: internals.passthruOn('name')
     },
+    {
+        place: 'event/definitions',
+        method: 'event',
+        list: true,
+        useFilename: internals.passthruOn('name')
```

Comment:
@devinivy I wondered if I should utilize `useFilename` in my first commit, but didn't really understand what its doing, or what benefit this brings.  Still don't, copied this from cookies -- seemed right.  Can you shed some light on it?